src/installation/guides/fullusb.md

```diff
@@ -0,0 +1,45 @@
+# Live USB To Full Install
+
+This installation guide shows how to convert a Live USB of Void Linux into a
+Full Pervasive Install. It's recommended to use a machine with at least 4GB of
+RAM, and a USB with at least 4GB of storage to perform the installation.
```

Comment:
I prefer not to use contractions (even if sometimes one can go through accidentaly :p). So "It is".

Reply:
"...4GB of RAM and a USB stick with..."